### PR TITLE
Change: disable mainnet connection

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

Disable mainnet connection on wCCD example

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: examples/wCCD/src/Root.tsx [hunk 1]
```diff
@@ -17,10 +17,26 @@ export default function Root() {
...
+
     useEffect(() => {
         detectConcordiumProvider()
             .then((provider) => {
                 // Listen for relevant events from the wallet.
+                provider.on('chainChanged', (genesisBlock) => {
+                    // Check if the user is connected to testnet by checking if the genesisBlock is the testnet one.
+                    // Throw a warning and disconnect if wrong chain. We only want to
+                    // allow users to interact with our testnet smart contracts.
+                    if (genesisBlock !== '4221332d34e1694168c2a0c0b3fd0f273809612cb13d000d5c2e00e85f50f796') {
```
Comment (abizjak) [inline examples/wCCD/src/Root.tsx:33]: Is it possible to put this in a named, documented constant so that it does not appear misteriously in two places?

**Review by abizjak — APPROVED** [review]
This looks reasonable to me, but @shjortConcordium and @orhoj have to be the final judge :)

**Review by shjortConcordium — APPROVED** [review]
LGTM, although it is annoying that you don't get a "network changed" or "account disconnected" when changing to mainnet while your mainnet account is not connected. :thinking:
(But I guess that is an issue with the wallet itself. Otherwise you would have to check what the network is each time the account changes, which might be okay if we had an endpoint for that, that didn't require the dApp to query the node :cry:)

**Comment by shjortConcordium** [comment]
> > LGTM, although it is annoying that you don't get a "network changed" or "account disconnected" when changing to mainnet while your mainnet account is not connected. thinking (But I guess that is an issue with the wallet itself. Otherwise you would have to check what the network is each time the account changes, which might be okay if we had a direct endpoint for that, one that didn't require the dApp to query the node cry)
> 
> Is such an API something that should be added to the wallet? If so, where does one record such desires?

I have added an issue in Jira.

**Pull request merged**